src/main/java/Duke.java

```diff
@@ -1,10 +1,232 @@
+    public static void changeStatus(Task task) {
+        printHorizontalLine();
+        System.out.println("Nice! I've marked this task as done:");
+
+        task.setIsDone(true);
+        System.out.println(task.toString());
+
+        System.out.println();
+        printHorizontalLine();
+    }
+
+    public static Task createType(String message) {
+        String[] infoEntered = message.split(" ");
+        int nameLength;
+        try {
+            switch (infoEntered[0]) {
+            case ("todo"):
+                nameLength = 4;
+                if (infoEntered.length == 1) {
+                    throw new DukeException();
+                }
+                return new ToDo(message.substring(nameLength + 1));
+            case ("deadline"):
+                nameLength = 8;
+                if (infoEntered.length == 1 || !message.contains("/by")) {
+                    throw new DukeException();
+                }
+                int indexBy = message.indexOf("/");
+                return new Deadline(message.substring(nameLength + 1, indexBy - 1),
+                        message.substring(indexBy + 4));
+            case ("event"):
+                nameLength = 5;
+                if (infoEntered.length == 1 || !message.contains("/at")) {
+                    throw new DukeException();
+                }
+                int indexAt = message.indexOf("/");
+                return new Event(message.substring(nameLength + 1, indexAt - 1),
+                        message.substring(indexAt + 4));
+            default:
+                printHorizontalLine();
+                System.out.println("☹ OOPS!!! I'm sorry, but I don't know what that means :-(\n");
+                printHorizontalLine();
+                return null;
+            }
+        } catch (DukeException e) {
+            printHorizontalLine();
+            System.out.println("☹ OOPS!!! The description cannot be empty or invalid.\n");
+            printHorizontalLine();
+            return null;
+        }
+    }
+
+    public static void deleteItems(ArrayList<Task> tasks, int index) {
+        printHorizontalLine();
+        System.out.println("Noted. I've removed this task: ");
+        System.out.println(tasks.get(index).toString());
+
+        tasks.remove(tasks.get(index));
+
+        System.out.println("Now you have " + tasks.size() + " items in the list.\n");
+        printHorizontalLine();
+    }
+
     public static void main(String[] args) {
+
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        loadFromFile(FILE_PATH);
+        Scanner in = new Scanner(System.in);
+
+        printWelcomeMessage();
+
+        String enteredMessage = in.nextLine();
+        while (!enteredMessage.equals("bye")) {
```

Comment:
Maybe you should double-check the use of constants and magic number here. Maybe improve a bit the level of abstractions in each if or else if statements.